velocity/src/main/java/co/aikar/commands/VelocityMessageFormatter.java

```diff
@@ -12,6 +12,6 @@ public VelocityMessageFormatter(TextColor... colors) {
     @Override
     @SuppressWarnings("deprecation")
     String format(TextColor color, String message) {
-        return ComponentSerializers.LEGACY.serialize(ComponentSerializers.LEGACY.deserialize(message).color(color));
+        return LegacyComponentSerializer.legacy().serialize(LegacyComponentSerializer.legacy().deserialize(message).color(color));
```

Comment:
Shouldn't you use `LegacyComponentSerializer.INSTANCE` like in [kyori.text sample](https://github.com/KyoriPowered/text#serializing-and-deserializing-components)?

Reply:
`LegacyComponentSerializer.INSTANCE` was deprecated with `LegacyComponentSerializer.legacyLinking()` becoming an option (which makes `LegacyComponentSerializer` act more like the BungeeCord chat API and CraftBukkit).